mail/notmuch/Portfile

```diff
@@ -51,9 +49,6 @@ post-patch {
 
 conflicts_build     ${name} xcbuild
 
-compiler.blacklist-append *gcc-3.* *gcc-4.* {clang < 300} llvm-g++-4.2
-compiler.whitelist clang macports-clang-5.0 
-
```

Comment:
Why are these being removed?

Reply:
Because I want to see where in the hosted builds possible errors occur.